.github/workflows/Lockfile.yml

```diff
@@ -12,9 +12,18 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     steps:
-      - uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
         with:
-          egress-policy: audit
+          disable-sudo: true
+          egress-policy: block
+          allowed-endpoints: >
+            github.com:443
+            objects.githubusercontent.com:443
+            raw.githubusercontent.com:443
+            repo.maven.apache.org:443
+            repo1.maven.org:443
+            sh.jbang.dev:443
```

Comment:
How are you getting this list of endpoints?

Reply:
In the summary (accessible at the link printed in the harden-runner step) they list all endpoints that were accessed. I used the "Recommended" tab where they contained the recommended config.

Reply:
https://app.stepsecurity.io/github/chains-project/maven-lockfile/actions/runs/11597128578